pennylane_lightning/core/src/bindings/Bindings.cpp

```diff
@@ -85,6 +85,6 @@ PYBIND11_MODULE(
     // Register bindings for backend-specific info:
     registerBackendSpecificInfo(m);
 
-    registerLightningTensorClassBindings<TensorNetBackends>(m);
+    registerLightningTensorClassBindings<AllTensorNetBackends>(m);
```

Comment:
as mentioned in bindings, can we call this TensorNetBackends?

Reply:
Done!